.pyup.yml

```diff
@@ -0,0 +1,8 @@
+schedule: "every day"
+search: False
+update: insecure
+requirements:
+  - requirements.txt:
+      update: security
```

Comment:
I think this isn't needed as they'll inherit from the `update` field on line 3. It also appears that the string `security` was a [typo in the docs](https://github.com/pyupio/pyup/blob/master/pyup/config.py#L27) and `insecure` is the correct value to use.